src/H5Eprivate.h

```diff
@@ -129,6 +130,35 @@ typedef struct H5E_t H5E_t;
         HGOTO_ERROR(majorcode, minorcode, retcode, "%s, errno = %d, error message = '%s'", str, myerrno,     \
```

Comment:
It's interesting that this does a 'goto' when the DONE_ macros are usually used past a goto label specifically. The fact that the new windows implementation works this way may cause some issues.

Reply:
I'm not sure what you mean here. The Win32 implementation is a copy of the POSIX one and in both cases there is a DONE <=> DONE and GOTO <=> GOTO correspondence.

Reply:
Is there a typo I'm not seeing?

Reply:
Ah, my mistake, ignore this and the other comment. Git's shortening of the diff made it appear as though the non-windows implementation for HSYS_DONE_ERROR used the HGOTO_ERROR from HSYS_GOTO_ERROR.